src/Arcus.Testing.Assert/Arcus.Testing.Assert.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <RootNamespace>Arcus.Testing</RootNamespace>
```

Comment:
Made this the root namespace for more easier access, and not to conflict with other testing frameworks (like xUnit/NUnit has a type `Assert`, and it would conflict with a namespace).